inference-engine/ie_bridges/python/sample/classification_sample_async/README.md

```diff
@@ -32,8 +32,8 @@ python3 classification_sample_async.py -h
 The command yields the following usage message:
```

Comment:
Is it correct name according changes? May be it should have `classification_sample` name?